doc/adrs/130_add_more_filter_capabilities_on_diagram.adoc

```diff
@@ -0,0 +1,50 @@
+= ADR-127 - Add more filter capabilities on diagram
+
+== Context
+
+Right now, we have severals tools to filter diagram elements.
+
+=== Nodes
+
+We can hide or fade a node.
+
+In the case where the node is a compartiments, we can collapse/uncollapse. 
```

Comment:
- there's no such thing as a compartment, this concept does not exist
- any node can be collapsed